src/components/location/dto/create-location.dto.ts

```diff
@@ -1,83 +1,41 @@
 import { Field, InputType, ObjectType } from '@nestjs/graphql';
 import { Type } from 'class-transformer';
 import { ValidateNested } from 'class-validator';
-import { IdField, NameField } from '../../../common';
-import { Country, Region, Zone } from './location.dto';
+import { IdField, NameField, Sensitivity } from '../../../common';
+import { LocationType } from './location-type.enum';
+import { Location } from './location.dto';
 
 @InputType()
-export abstract class CreateZone {
+export abstract class CreateLocation {
   @NameField()
   readonly name: string;
 
-  @IdField({
-    description: 'A user ID that will be the director of the zone',
-  })
-  readonly directorId: string;
-}
+  @Field(() => LocationType)
+  readonly type: LocationType;
 
-@InputType()
-export abstract class CreateRegion {
-  @NameField()
-  readonly name: string;
-
-  @IdField({
-    description: 'The zone ID that the region will be associated with',
-  })
-  readonly zoneId: string;
-
-  @IdField({
-    description: 'A user ID that will be the director of the region',
-  })
-  readonly directorId: string;
-}
+  @Field(() => Sensitivity)
+  readonly sensitivity: Sensitivity;
 
-@InputType()
-export abstract class CreateCountry {
-  @NameField()
-  readonly name: string;
+  @Field({ nullable: true })
+  readonly iso31663?: string;
 
-  @IdField()
-  readonly regionId: string;
-}
+  @NameField({ nullable: true })
+  readonly geographyName?: string;
```

Comment:
This is just the name associated with the ISO-3166-3 code. I'd rather not manage input this separately. If we need to provide it we should look it up with the code. I'm comfortable with this happening later.